markets/perps-market/contracts/modules/LiquidationModule.sol

```diff
@@ -18,28 +18,22 @@ contract LiquidationModule is ILiquidationModule {
     }
 
     function liquidateFlagged() external override {
-        PerpsMarketFactory.Data storage factory = PerpsMarketFactory.load();
-        SetUtil.UintSet storage liquidatableAccounts = factory.liquidatableAccounts;
+        SetUtil.UintSet storage liquidatableAccounts = GlobalPerpsMarket
+            .load()
+            .liquidatableAccounts;
 
-        for (uint i = 0; i < factory.liquidatableAccounts.length(); i++) {
+        for (uint i = 0; i < liquidatableAccounts.length(); i++) {
             _liquidateAccount(liquidatableAccounts.valueAt(i).to128());
         }
     }
 
-    function _liquidateAccount(uint128 accountId) private {
+    function _liquidateAccount(uint128 accountId) internal {
```

Comment:
Is there a way to just flag and not liquidate? 
If I understand the behavior correctly, it will check if the account can be flagged for liquidation, mark it (flag) and liquidate. or revert. So flag and liquidate is in fact a single step.

Reply:
we will probably want to add another external function to flag account